rust/operator/src/operations/graceful_shutdown.rs

```diff
@@ -0,0 +1,12 @@
+use stackable_hdfs_crd::MergedConfig;
+use stackable_operator::builder::PodBuilder;
+
+pub fn add_graceful_shutdown_config(
+    merged_config: &(dyn MergedConfig + Send + 'static),
+    pod_builder: &mut PodBuilder,
+) {
+    // This must be always set by the merge mechanism, as we provide a default value.
+    if let Some(graceful_shutdown_timeout) = merged_config.graceful_shutdown_timeout() {
+        pod_builder.termination_grace_period_seconds(graceful_shutdown_timeout.as_secs() as i64);
```

Comment:
```suggestion
        pod_builder.termination_grace_period_seconds(graceful_shutdown_timeout.as_secs().try_into().context(DurationTooLongSnafu { duration })?);
```

`foo as i64` will silently turn numbers above `9223372036854775808` negative. It's very much a corner-case here (anyone setting timers around over 200 billion years into the future has probably got bigger issues...), but I'd rather just blanket-ban the syntax (`#![deny(clippy::as_conversions)]` or even `#![deny(clippy::pedantic)]`...).